packages/component/src/Attachment/DownloadAttachment.js

```diff
@@ -28,7 +29,7 @@ export default connectToWebChat(
                 <div className="size">{ format(size) }</div>
             }
           </div>
-          <DownloadIcon className="icon" size={ 1.5 } />
+          <DownloadIcon className="icon" language={ language } size={ 1.5 } />
```

Comment:
instead of passing the language and letting `<DownloadIcon />` do the localization, could we just add a `label` prop and have `<DownloadAttachment />` pass in the already localized string? I like the separation there a bit better.

Reply:
👍